src/main/java/com/google/neighborgood/servlets/TaskServlet.java

```diff
@@ -188,6 +191,21 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
       return;
     }
 
+    // Get the task overview from the form input
+    String taskOverview = "";
+    input = request.getParameter("task-overview-input");
+    // If the input is valid, set the taskOverview value to the input value
+    if (input != null) {
+      taskOverview = input.trim();
+    }
+
+    // If input task overview is empty, reject the request to add a new task and send a 400 error.
+    if (taskOverview.equals("")) {
+      System.err.println("The input task overview is empty");
+      response.sendRedirect("/400.html");
```

Comment:
As as above

Reply:
Okay, I've created issue #84 for the 400 redirects removal.